app/src/main/java/net/gsantner/markor/format/markdown/ListHandler.java

```diff
@@ -38,8 +48,12 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
                 sSpan.setSpan(this, iStart, start + 1, Spanned.SPAN_COMPOSING);
             } else {
                 Matcher oMatch = MarkdownHighlighterPattern.LIST_ORDERED.pattern.matcher(previousLine);
-                if (oMatch.find() && previousLine.equals(oMatch.group(1) + ". ")) {
-                    sSpan.setSpan(this, iStart, start + 1, Spanned.SPAN_COMPOSING);
+                if (oMatch.find()) {
+                    if (previousLine.equals(oMatch.group(1) + ". ")) {
```

Comment:
Do I see right, this is the only relevant point for ordered list on each line (ie  1. text\n2. text\3. text)? If so, please make `". "` a `final String ul_sep = ". ";` in the methods beginning. This makes it easier to see next time when coming here.... where the most important point is.

Reply:
I'll be fixing this in the next PR :) where I update the way Listhandler detects empty lines among other things.